website/docs/reference/datasources/hubspot/README.md

```diff
@@ -0,0 +1,152 @@
+
+# HubSpot
+
+
+HubSpot is a **cloud-based CRM** designed to help align sales and marketing teams, promote sales enablement, increase ROI, and improve your inbound marketing strategy to produce more qualified leads. In simple words, it is a software platform created to assist your business with marketing and sales.
+
+:::info
+To integrate Hubspot with Appsmith, you should understand the [basics of connecting to databases](core-concepts/connecting-to-data-sources/) on Appsmith.
+:::
+
+Appsmith provides an easy way to integrate with HubSpot, and enables you to perform different operations using a custom UI built with minimal configurations.
+
+
+# Create HubSpot Datasource
+
+To add a HubSpot API, navigate to **Explorer** >> Click on the **+** icon (next to Datasources) >> Select **HubSpot API** under APIs.
```

Comment:
The current version of app.appsmith.com shows the name `HubSpot` and doesn't have an API. Could you please verify this and change it accordingly?